### ISSUE TITLE
Loss Convergance

### ISSUE THREAD

**Issue by DYJNG** [MRzzm/DINet#59]
When Loss_perception value is what, can we consider the model to be convergent?

**Comment by 1129571** [comment]
I also have the same question
我也有同样的疑问

**Comment by tailangjun** [comment]
64都感觉好慢，256更不敢想象，不加个断点续训的 load_checkoutpoints是不行了
===> Epoch[46](82/101):  Loss_DI: 0.2182 Loss_GI: 0.2981 Loss_perception: 2.5939 lr_g = 0.0001000 
===> Epoch[46](83/101):  Loss_DI: 0.2150 Loss_GI: 0.3149 Loss_perception: 2.7463 lr_g = 0.0001000 
===> Epoch[46](84/101):  Loss_DI: 0.2148 Loss_GI: 0.2981 Loss_perception: 2.6907 lr_g = 0.0001000 
===> Epoch[46](85/101):  Loss_DI: 0.2152 Loss_GI: 0.2994 Loss_perception: 2.6515 lr_g = 0.0001000 
===> Epoch[46](86/101):  Loss_DI: 0.2145 Loss_GI: 0.3324 Loss_perception: 2.6788 lr_g = 0.0001000 
===> Epoch[46](87/101):  Loss_DI: 0.2184 Loss_GI: 0.3122 Loss_perception: 2.6443 lr_g = 0.0001000 
===> Epoch[46](88/101):  Loss_DI: 0.2185 Loss_GI: 0.2812 Loss_perception: 2.7179 lr_g = 0.0001000 
===> Epoch[46](89/101):  Loss_DI: 0.2226 Loss_GI: 0.2833 Loss_perception: 2.6480 lr_g = 0.0001000 
===> Epoch[46](90/101):  Loss_DI: 0.2204 Loss_GI: 0.3086 Loss_perception: 2.6684 lr_g = 0.0001000 
===> Epoch[46](91/101):  Loss_DI: 0.2240 Loss_GI: 0.3178 Loss_perception: 2.6519 lr_g = 0.0001000 
===> Epoch[46](92/101):  Loss_DI: 0.2230 Loss_GI: 0.2898 Loss_perception: 2.6223 lr_g = 0.0001000 
===> Epoch[46](93/101):  Loss_DI: 0.2211 Loss_GI: 0.2988 Loss_perception: 2.6694 lr_g = 0.0001000 
===> Epoch[46](94/101):  Loss_DI: 0.2192 Loss_GI: 0.3032 Loss_perception: 2.6445 lr_g = 0.0001000 
===> Epoch[46](95/101):  Loss_DI: 0.2214 Loss_GI: 0.2961 Loss_perception: 2.6282 lr_g = 0.0001000 
===> Epoch[46](96/101):  Loss_DI: 0.2179 Loss_GI: 0.2983 Loss_perception: 2.6242 lr_g = 0.0001000 
===> Epoch[46](97/101):  Loss_DI: 0.2136 Loss_GI: 0.3454 Loss_perception: 2.6110 lr_g = 0.0001000 
===> Epoch[46](98/101):  Loss_DI: 0.2147 Loss_GI: 0.3127 Loss_perception: 2.6875 lr_g = 0.0001000 
===> Epoch[46](99/101):  Loss_DI: 0.2166 Loss_GI: 0.2903 Loss_perception: 2.6161 lr_g = 0.0001000 
===> Epoch[46](100/101):  Loss_DI: 0.2296 Loss_GI: 0.3108 Loss_perception: 2.7418 lr_g = 0.0001000

**Comment by lililuya** [comment]
> 64都感觉好慢，256更不敢想象，不加个断点续训的 load_checkoutpoints是不行了 ===> Epoch[46](82/101): Loss_DI: 0.2182 Loss_GI: 0.2981 Loss_perception: 2.5939 lr_g = 0.0001000 ===> Epoch[46](83/101): Loss_DI: 0.2150 Loss_GI: 0.3149 Loss_perception: 2.7463 lr_g = 0.0001000 ===> Epoch[46](84/101): Loss_DI: 0.2148 Loss_GI: 0.2981 Loss_perception: 2.6907 lr_g = 0.0001000 ===> Epoch[46](85/101): Loss_DI: 0.2152 Loss_GI: 0.2994 Loss_perception: 2.6515 lr_g = 0.0001000 ===> Epoch[46](86/101): Loss_DI: 0.2145 Loss_GI: 0.3324 Loss_perception: 2.6788 lr_g = 0.0001000 ===> Epoch[46](87/101): Loss_DI: 0.2184 Loss_GI: 0.3122 Loss_perception: 2.6443 lr_g = 0.0001000 ===> Epoch[46](88/101): Loss_DI: 0.2185 Loss_GI: 0.2812 Loss_perception: 2.7179 lr_g = 0.0001000 ===> Epoch[46](89/101): Loss_DI: 0.2226 Loss_GI: 0.2833 Loss_perception: 2.6480 lr_g = 0.0001000 ===> Epoch[46](90/101): Loss_DI: 0.2204 Loss_GI: 0.3086 Loss_perception: 2.6684 lr_g = 0.0001000 ===> Epoch[46](91/101): Loss_DI: 0.2240 Loss_GI: 0.3178 Loss_perception: 2.6519 lr_g = 0.0001000 ===> Epoch[46](92/101): Loss_DI: 0.2230 Loss_GI: 0.2898 Loss_perception: 2.6223 lr_g = 0.0001000 ===> Epoch[46](93/101): Loss_DI: 0.2211 Loss_GI: 0.2988 Loss_perception: 2.6694 lr_g = 0.0001000 ===> Epoch[46](94/101): Loss_DI: 0.2192 Loss_GI: 0.3032 Loss_perception: 2.6445 lr_g = 0.0001000 ===> Epoch[46](95/101): Loss_DI: 0.2214 Loss_GI: 0.2961 Loss_perception: 2.6282 lr_g = 0.0001000 ===> Epoch[46](96/101): Loss_DI: 0.2179 Loss_GI: 0.2983 Loss_perception: 2.6242 lr_g = 0.0001000 ===> Epoch[46](97/101): Loss_DI: 0.2136 Loss_GI: 0.3454 Loss_perception: 2.6110 lr_g = 0.0001000 ===> Epoch[46](98/101): Loss_DI: 0.2147 Loss_GI: 0.3127 Loss_perception: 2.6875 lr_g = 0.0001000 ===> Epoch[46](99/101): Loss_DI: 0.2166 Loss_GI: 0.2903 Loss_perception: 2.6161 lr_g = 0.0001000 ===> Epoch[46](100/101): Loss_DI: 0.2296 Loss_GI: 0.3108 Loss_perception: 2.7418 lr_g = 0.0001000

Hi，你用的是多大的数据集训的，我使用自己的4K的人脸数据训练，但是loss在4左右就无法下降了，根据数据量我调整了learning_rate 的decay时机

**Comment by tailangjun** [comment]
> > 64都感觉好慢，256更不敢想象，不加个断点续训的 load_checkoutpoints是不行了 ===> Epoch[46](82/101): Loss_DI: 0.2182 Loss_GI: 0.2981 Loss_perception: 2.5939 lr_g = 0.0001000 ===> Epoch[46](83/101): Loss_DI: 0.2150 Loss_GI: 0.3149 Loss_perception: 2.7463 lr_g = 0.0001000 ===> Epoch[46](84/101): Loss_DI: 0.2148 Loss_GI: 0.2981 Loss_perception: 2.6907 lr_g = 0.0001000 ===> Epoch[46](85/101): Loss_DI: 0.2152 Loss_GI: 0.2994 Loss_perception: 2.6515 lr_g = 0.0001000 ===> Epoch[46](86/101): Loss_DI: 0.2145 Loss_GI: 0.3324 Loss_perception: 2.6788 lr_g = 0.0001000 ===> Epoch[46](87/101): Loss_DI: 0.2184 Loss_GI: 0.3122 Loss_perception: 2.6443 lr_g = 0.0001000 ===> Epoch[46](88/101): Loss_DI: 0.2185 Loss_GI: 0.2812 Loss_perception: 2.7179 lr_g = 0.0001000 ===> Epoch[46](89/101): Loss_DI: 0.2226 Loss_GI: 0.2833 Loss_perception: 2.6480 lr_g = 0.0001000 ===> Epoch[46](90/101): Loss_DI: 0.2204 Loss_GI: 0.3086 Loss_perception: 2.6684 lr_g = 0.0001000 ===> Epoch[46](91/101): Loss_DI: 0.2240 Loss_GI: 0.3178 Loss_perception: 2.6519 lr_g = 0.0001000 ===> Epoch[46](92/101): Loss_DI: 0.2230 Loss_GI: 0.2898 Loss_perception: 2.6223 lr_g = 0.0001000 ===> Epoch[46](93/101): Loss_DI: 0.2211 Loss_GI: 0.2988 Loss_perception: 2.6694 lr_g = 0.0001000 ===> Epoch[46](94/101): Loss_DI: 0.2192 Loss_GI: 0.3032 Loss_perception: 2.6445 lr_g = 0.0001000 ===> Epoch[46](95/101): Loss_DI: 0.2214 Loss_GI: 0.2961 Loss_perception: 2.6282 lr_g = 0.0001000 ===> Epoch[46](96/101): Loss_DI: 0.2179 Loss_GI: 0.2983 Loss_perception: 2.6242 lr_g = 0.0001000 ===> Epoch[46](97/101): Loss_DI: 0.2136 Loss_GI: 0.3454 Loss_perception: 2.6110 lr_g = 0.0001000 ===> Epoch[46](98/101): Loss_DI: 0.2147 Loss_GI: 0.3127 Loss_perception: 2.6875 lr_g = 0.0001000 ===> Epoch[46](99/101): Loss_DI: 0.2166 Loss_GI: 0.2903 Loss_perception: 2.6161 lr_g = 0.0001000 ===> Epoch[46](100/101): Loss_DI: 0.2296 Loss_GI: 0.3108 Loss_perception: 2.7418 lr_g = 0.0001000
> 
> Hi，你用的是多大的数据集训的，我使用自己的4K的人脸数据训练，但是loss在4左右就无法下降了，根据数据量我调整了learning_rate 的decay时机

我当时只抽取了200条视频，目的是为了把整个流程走通
